test/e2e/examples.go

```diff
@@ -157,6 +157,47 @@ var _ = Describe("Examples e2e", func() {
 			}
 		})
 	})
+
+	Describe("[Skipped][Example]RethinkDB", func() {
+		It("should create and stop rethinkdb servers", func() {
+			mkpath := func(file string) string {
+				return filepath.Join(testContext.RepoRoot, "examples", "rethinkdb", file)
+			}
+			driverServiceYaml := mkpath("driver-service.yaml")
+			rethinkDbControllerYaml := mkpath("rc.yaml")
+			adminPodYaml := mkpath("admin-pod.yaml")
+			adminServiceYaml := mkpath("admin-service.yaml")
+			nsFlag := fmt.Sprintf("--namespace=%v", ns)
+
+			By("starting rethinkdb")
+			runKubectl("create", "-f", driverServiceYaml, nsFlag)
+			runKubectl("create", "-f", rethinkDbControllerYaml, nsFlag)
+			checkDbInstances := func() {
+				forEachPod(c, ns, "db", "rethinkdb", func(pod api.Pod) {
+					_, err := lookForStringInLog(ns, pod.Name, "rethinkdb", "Server ready", serverStartTimeout)
+					Expect(err).NotTo(HaveOccurred())
+				})
+			}
+			checkDbInstances()
+
+			By("scaling rethinkdb")
+			ScaleRC(c, ns, "rethinkdb-rc", 2)
```

Comment:
I think it's OK to leave it as it is, but I'm slightly worried that hardcoding number of replicas here may cause some problems in the future. If someone would change the default replica count in the example, tests won't catch this change. It'd be cleaner to have an util method "AddOneReplica", or check the replica count beforehand.